### PR TITLE
Update base-fr.yaml

### PR DESCRIPTION
**Pull request by Naheulf**
Fix typographic error, grammar error and few other errors.
Most typographic errors fixed:
- Unbreakable space before "!", ":" and units.
- Typographic french apostrophe (’) instead of english one (')
- Multiplication sign (×) instead of letter x.
- Fraction sign ( ⁄ )instead of slash (/)
- "Tiret cadratin" ( — ) instead of normal "tiret" to separate sentences.

### CODE REVIEW

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -510,37 +511,37 @@ buildings:
...
     trash:
         default:
             name: &trash Poubelle
-            description: Accepte des formes de n'importe quel côté et les détruit... pour toujours.
+            description: Accepte des formes de n’importe quel côté et les détruit… pour toujours.
 
         storage:
             name: Stockage
-            description: Stocke les formes en trop jusqu'à une certaine capacité. Peut être utilisé comme tampon.
+            description: Stocke les formes en trop jusqu…à une certaine capacité. Peut être utilisé comme tampon.
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:526]: I think you wanted to make an apostrophe here

File: translations/base-fr.yaml [hunk 2]
```diff
@@ -817,15 +818,15 @@ keybindings:
...
+        placeInverse: Inverser le mode d’orientation automatique
         pasteLastBlueprint: Copier le dernier patron
         massSelectCut: Couper la sélection
-        exportScreenshot: Exporter toute la base en tant qu'image.
+        exportScreenshot: Exporter toute la base en tant qu’image.
         mapMoveFaster: Se déplacer plus vite
-        lockBeltDirection: Utiliser le plannificateur de convoyeurs
-        switchDirectionLockSide: "Plannificateur: changer de côté"
+        lockBeltDirection: Utiliser le planificateur de convoyeurs
+        switchDirectionLockSide: "Planificateur: changer de côté"
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:829]: Space

**Review by DerFeldspatz — APPROVED** [review]
Changes look clean and consistent to me

File: translations/base-fr.yaml [hunk 3]
```diff
@@ -82,14 +82,15 @@ steamPage:
             [*] [url=https://github.com/tobspr/shapez.io/blob/master/translations/README.md]Aidez à traduire[/url]
         [/list]
 
-    discordLink: Discord officiel - Parlez avec moi!
+    discordLink: Discord officiel — Parlez avec moi!
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:85]: Space

File: translations/base-fr.yaml [hunk 4]
```diff
@@ -387,19 +388,19 @@ ingame:
...
+            1_1_extractor: Placez un <strong>extracteur</strong> sur une <strong>forme en cercle</strong> pour l’extraire !
             1_2_conveyor: >-
-                Connectez l'extracteur avec un <strong>convoyeur</strong> vers votre centre !<br><br>Astuce: <strong>Cliquez et faites glisser</strong> le convoyeur avec votre souris !
+                Connectez l’extracteur avec un <strong>convoyeur</strong> vers votre centre !<br><br>Astuce : <strong>Cliquez et faites glisser</strong> le convoyeur avec votre souris !
 
             1_3_expand: >-
-                Ceci n'est <strong>PAS</strong> un jeu incrémental et inactif ! Construisez plus d'extracteurs et de convoyeurs pour atteindre plus vite votre votre but.<br><br>Astuce: Gardez <strong>MAJ</strong> enfoncé pour placer plusieurs extracteurs, et utilisez <strong>R</strong> pour les faire pivoter.
+                Ceci n’est <strong>PAS</strong> un jeu incrémental et inactif ! Construisez plus d extracteurs et de convoyeurs pour atteindre plus vite votre but.<br><br>Astuce : Gardez <strong>MAJ</strong> enfoncé pour placer plusieurs extracteurs, et utilisez <strong>R</strong> pour les faire pivoter.
```
Comment (DerFeldspatz) [inline translations/base-fr.yaml:403]: One space should be an apostrophe

**Review by DerFeldspatz — CHANGES_REQUESTED** [review]
I think you are missing 1 more space in line 812

File: translations/base-fr.yaml (hunk 1 above)
Comment (Naheulf) [reply translations/base-fr.yaml:526]: Yes. Bad copy/paste.

File: translations/base-fr.yaml (hunk 2 above)
Comment (Naheulf) [reply translations/base-fr.yaml:829]: Yes; I missed it.

File: translations/base-fr.yaml (hunk 3 above)
Comment (Naheulf) [reply translations/base-fr.yaml:85]: Fixed

File: translations/base-fr.yaml (hunk 4 above)
Comment (Naheulf) [reply translations/base-fr.yaml]: Yes. Wrong copy-paste

**Comment by DerFeldspatz** [comment]
I think you are still missing 1 more space in line 812

**Comment by LeopoldTal** [comment]
Will conflict with #629, suggest we merge this into a single PR. (I also loathe most of these typographic choices with the burning passion of a thousand suns, but eh, it's up to tobspr.)

**Comment by Naheulf** [comment]
I'll recreate the patch from https://github.com/LeopoldTal/shapez.io/tree/french-pedantry

**Comment by DerFeldspatz** [comment]
You can always make propositions to the translation, even the typographic choices if it fits your language better. There is no need to use the same as tobspr. He has to start somewhere and you can always provide better solutions. If he pulls them is up to him.

**Comment by Naheulf** [comment]
The issue is that I miss to check for existing PR before create mine. Since there already another one that edit the same language file the two PR are unable to automatically merge together. So, a native french speaker (LeopoldTal or me or another people) have to merge them.
My PR come after and make few changes that also apply LeopoldTal ones. So it come down to me to merge the two. And, sine I was unable to merge them in one step on my PR without wiping-out LeopoldTal from commit history, i choose to apply the merged file to the LeopoldTal repo (and by the way to his PR).

I hope my message is understandable since it's the first long message I wrote without help of translator.

**Pull request closed without merging**